pkg/utils/common.go

```diff
@@ -594,16 +596,15 @@ func PrintUnreportedErrors(
 
 func NewLogger(
 	logName string,
-	logLevelStr string,
+	sigAggFlags flags.SignatureAggregatorFlags,
 	defaultLogLevelStr string,
 	logDir string,
-	logToStdout bool,
 	print func(string, ...interface{}),
 ) (logging.Logger, error) {
-	logLevel, err := logging.ToLevel(logLevelStr)
+	logLevel, err := logging.ToLevel(sigAggFlags.AggregatorLogLevel)
 	if err != nil {
-		if logLevelStr != "" {
-			print("undefined logLevel %s. Setting %s log to %s", logLevelStr, logName, defaultLogLevelStr)
+		if sigAggFlags.AggregatorLogLevel != "" {
```

Comment:
Do we need this check if we're already validating in the flag creation?

Reply:
Good catch. technically not since we are already setting default and already have the validate function